### ISSUE TITLE
Lists farther to the right missing header card styling

### ISSUE THREAD

**Issue by jsheise** [beingmrkenny/ListHighlighterForTrello#6]
Hey, all!

## Issue Summary
Not sure if anybody else has encountered this issue, though I figure it cannot hurt to start a thread on it before I go digging into the repo!

I've noticed for a while that the **formatting for header cards in Trello List Highlighter does not consistently apply to my lists**.

After noticing during a board reorganization that it would apply to some lists and not others, I decided to dig into it some more and found that it **seems to only occur on lists further to the right. Since those are lists that would be loaded later, I suspect that it has to do with some other script interrupting TLH's sequence of events on page load**.

This occurs consistently in my everyday board, though the replications below were snipped out of multiple page refreshes. In other words, on my "Demo" board, it was hit-or-miss on every page refresh.

## Demonstrations - Page Reload
I found that depending on where I placed a list from left-to-right in my personal board, the styling would get applied. For example, if I moved my "PERSONAL GOALS" list from position 10 to position 13, the header card styles would not get applied like they would when the list was in position 10.

I initially figured it might have had to do with the specific number of lists, so I created a demo board with 14 lists, 6 cards to each list.

### Successful styling on all lists

https://github.com/beingmrkenny/ListHighlighterForTrello/assets/41879444/05017d6e-a628-4ea6-abb6-9c7e075ae29f

Console output:
[console-export-2023-9-23_18-6-44 - none affected.txt](https://github.com/beingmrkenny/ListHighlighterForTrello/files/12707526/console-export-2023-9-23_18-6-44.-.none.affected.txt)

### Styling missing on list 14

https://github.com/beingmrkenny/ListHighlighterForTrello/assets/41879444/9782b5a7-efa9-4bd3-bee0-b2da4be43f72

Console output:
[console-export-2023-9-23_18-6-57 - 14 affected - 3.txt](https://github.com/beingmrkenny/ListHighlighterForTrello/files/12707525/console-export-2023-9-23_18-6-57.-.14.affected.-.3.txt)

### Styling missing on lists 12, 13, 14

https://github.com/beingmrkenny/ListHighlighterForTrello/assets/41879444/6dfc0714-c107-4eb4-864a-7decd3690736

Console output:
[console-export-2023-9-23_18-6-28 - 12,13,14 affected.txt](https://github.com/beingmrkenny/ListHighlighterForTrello/files/12707524/console-export-2023-9-23_18-6-28.-.12.13.14.affected.txt)

I could not find any significant differences when diffing the console logs (attached for each) between a successful load and an unsuccessful load. 

## Demonstration - Editing a card via card back (not card front) triggers styling
I assume this has to do with the events that TLH is listening for, though maybe somebody can clarify why only updating via the card back works? Is it because we are updating via a separate element entirely versus whatever div or span contains the card title?

### Replication steps:

1. Find a header card with missing styling. In my case, this tends to be on a list in the 12+ position.
2. Edit the title via card front. This does not cause the styling to be applied.
3. Open the card back view
4. Edit the title and close the card back. Header card styling should be applied.
5. Refresh the page. The styling that was just applied is lost again.

https://github.com/beingmrkenny/ListHighlighterForTrello/assets/41879444/fd335c1f-fe44-4fe6-b536-61832fdb4f3c
(blurred because this was from my personal board)

---
I am on Firefox 117.0.1 (64-bit). If need be, I can try replicating in Chrome on my work laptop if that would be helpful, though I have seen it in Firefox on that machine as well. 
Clearing the browser cache does not seem to change any behavior.

Hopefully somebody smarter than me can figure it out easily, or knows whether this is a resolvable issue.

Thanks!

**Comment by jsheise** [comment]
Forgot to add, in the demo board shown above, there were no power-ups active on that board, and I disabled my other Trello addons for testing.

![nopowerups](https://github.com/beingmrkenny/ListHighlighterForTrello/assets/41879444/0b8c25d0-9045-45ad-8b49-91b90ace180f)
![disabled addons](https://github.com/beingmrkenny/ListHighlighterForTrello/assets/41879444/85fd580a-3c33-4665-88fc-9661a7eec149)

Also, FWIW, these are the settings I'm using:
![tlh_settings](https://github.com/beingmrkenny/ListHighlighterForTrello/assets/41879444/75e91047-5326-4ca8-a441-ef7bc4416e9b)

**Comment by beingmrkenny** [comment]
Thank you so much for such a detailed report. Trello made some changes to its interface which broke a lot of extensions. I have been able to fix mine, but with apologies, I have decided to remove header and separator cards from the extension. It was a very buggy feature which I could never fix, plus Trello has a native feature called Card Covers, which I think is better.

Bit more information at https://beingmrkenny.co.uk/web-extensions/list-highlighter-trello/version-5
